src/scheduler.h

```diff
@@ -62,8 +61,6 @@ class CScheduler
      */
     void MockForward(std::chrono::seconds delta_seconds);
 
-    // To keep things as simple as possible, there is no unschedule.
```

Comment:
Hmm, I kind-of think that comment's helpful.

Reply:
This should be clear from our general policy that everything should be as minimal as possible and that unused features should not be implemented.

If the comment is brought back, I'd prefer the alternative version:

```cpp
     // To keep things as simple as possible, there is no unschedule and the scheduler can not schedule events on your electronic barista